backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ##################################################################################
-# BASE ###########################################################################
+# BASE (Is pushed to DockerHub for rebranding) ###################################
 ##################################################################################
 FROM node:12.19.0-alpine3.10 as base
 
```

Comment:
:point_down: Remove all following build arguments. If you have to use them, don't use defaults because you want to force developers to assign build arguments. You never want null-substitutions like `1970-01-01T00:00:00.00Z` in your docker image.

Reply:
We do understand, but there are some things speaking against it, because we will not force developers to assign those build arguments.